packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -247,7 +247,7 @@ class _RawVectorGraphicsWidget extends SingleChildRenderObjectWidget {
   }
 }
 
-class _RenderVectorGraphics extends RenderProxyBox {
+class _RenderVectorGraphics extends RenderBox {
```

Comment:
what's different about this from renderproxybox now?

Reply:
some of the layout behaviors work a little differently, this is intended to work the same as flutter_svg